inspirehep/modules/theme/static/scss/format/detailed-record.scss

```diff
@@ -466,3 +466,107 @@ $darken: 8%;
 #citations div.dataTables_filter input, #references div.dataTables_filter input {
     max-width: inherit;
 }
+
+#record-detailed-jobs > .row {
+
+    &#record-detailed-jobs-information-deadline {
+        margin-bottom: 50px;
+
+        #record-detailed-jobs-information, #record-detailed-jobs-deadline {
+            background-color: white;
+        }
+    }
+
+    .detailed-jobs-information {
+        padding-top: 5px;
+        padding-bottom: 5px;
+        overflow: hidden;
+    }
+
+    #jobs-detailed-header-top {
+        padding-left: 20px;
+        padding-right: 20px;
+    }
+
+    #detailed-jobs-deadline-alert {
+        margin-left: 20px;
+        margin-right: 20px;       
+    }
+    
+    #record-detailed-similar-jobs {
+        & :nth-child(2) {
+            padding-bottom: 20px;
+            border-bottom: 1px solid gray;
+        }
+
+        & :nth-child(3) {
+            padding-top: 10px;
+        }
+    }
+
+    #similar-jobs {
+        margin-top: 10px;
+        margin-bottom: 10px;
+    }
+
+    #record-detailed-jobs-description,
+    #record-detailed-jobs-applications,
+    #record-detailed-similar-jobs {
+        background-color: white;
+    }
+
+    @media (min-width: $screen-md-min) {
+
+        #record-detailed-jobs-information,
+        #record-detailed-jobs-deadline {
+            min-height: 375px;
+        }
+
+        #record-detailed-jobs-description.col-md-8,
+        #record-detailed-jobs-information.col-md-8 {
+            width: 73%;
+            padding: 0;
+        }
+
+        #record-detailed-jobs-description {
+            min-height: 500px;
+        }
+    }
+
+    @media (max-width: $screen-md-min) {
+        #record-detailed-jobs-information {
+            margin-bottom: 50px;
+        }
+    }
+
+    #job-remove-this-listing {
+        padding: 10px;
+        padding-left: 20px;
+        background-color: #E6E6E6;
```

Comment:
Yes, would be good to use one of https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/theme/static/scss/base/variables.scss (or add this one to the `variables` file)